### PR TITLE
server: add a RoutePrefix option

### PR DESCRIPTION
**Pull request by creachadair**
In some cases it is useful to run TailSQL on a server that already has other
endpoints. To support this, add a RoutePrefix option: When this is set, it will
be prepended to the the UI, API, and static file routes handled by the tailsql
server.

- Add tests to exercise prefixed routes.
- Plumb the prefix into the UI template and use it for static files.
- Fix an absolute path in the style sheet.
- Fix an absolute path in the CSV download script.
- Log a prefixed route in the example CLI.

### CODE REVIEW

File: server/tailsql/static/style.css [hunk 1]
```diff
@@ -186,7 +186,7 @@ div.action {
 .action .ctrl:before {
   display: none;
 }
-.action .ctrl:-webkit-details-marker {
+.action .ctrl::-webkit-details-marker {
```
Comment (creachadair) [inline server/tailsql/static/style.css:189]: This is an unrelated syntax issue I just happened to notice while testing.

**Review by maisem — APPROVED** [review]
